Dockerfile

```
@@ -31,9 +31,8 @@ RUN go build -o rudder-cli ./cmd/rudder-cli/
 
 FROM alpine:${ALPINE_VERSION}
 
-RUN apk update && apk add tzdata
-RUN apk -U --no-cache upgrade && \
-    apk add --no-cache ca-certificates postgresql-client curl bash
+RUN apk --no-cache upgrade && \
```

It is a good idea to always do an update right before doing an upgrade or add command. That way the command will install the latest available packages. [link](https://wiki.alpinelinux.org/wiki/Alpine_Package_Keeper#:~:text=Tip%3A%20With%20remote%20repositories%2C%20it%20is%20a%20good%20idea%20to%20always%20do%20an%20update%20right%20before%20doing%20an%20upgrade%20or%20add%20command.%20That%20way%20the%20command%20will%20install%20the%20latest%20available%20packages.)
```suggestion
RUN apk --no-cache update && apk --no-cache upgrade && \
```

---

Not exactly, `--no-cache` will handle that for you, and leave no temporary files.

---

the request was to run `apk --no-cache update` before `apk --no-cache upgrade`, not sure how your response is relevant

---

`--no-cache` actually will make `apk` ignore any local cache and fetch the latest index without local temp files needed to be cleaned up